python/lsst/meas/algorithms/testUtils.py

```diff
@@ -178,3 +184,116 @@ def makeDefectList():
                   ]
 
     return defectList
+
+
+class MockLoadReferenceObjects(ReferenceObjectLoader):
+    """A simple mock of LoadReferenceObjects for tests.
+
+    This class will read in files and return subsets to mock the behavior
+    of LoadReferenceObjects for tests.
+
+    Parameters
+    ----------
+    filenames : `list` [`str`]
+        Names of files to load.
+    config : `lsst.meas.astrom.LoadReferenceObjectsConfig`
+        Configuration object if necessary to override defaults.
+    convert : `bool`
+        Convert from old refcat format?
+    """
+    def __init__(self, filenames, config=None, convert=False):
+        self._cat = self._loadFiles(filenames, convert=convert)
+        self.config = config
+        if self.config is None:
+            self.config = LoadReferenceObjectsConfig()
+
+    def loadSkyCircle(self, ctrCoord, radius, filterName, epoch=None):
```

Comment:
Should we mention that the base class returns a SourceCatalog but this version returns a Struct? Am I reading the wrong documentation?

Reply:
You are reading the documentation correctly, but the documentation was wrong. I have now fixed it on this ticket.